reflex-algos/src/main/python/mlops/parallelm/mlops/mlops.py

```diff
@@ -930,6 +932,55 @@ def load_time_capture(self, input_file):
         from parallelm.mlops.time_capture.time_capture import TimeCapture
         return TimeCapture(input_file)
 
+    def attach(self,
+               ion_id,
+               mlops_server=Constants.MLOPS_DEFAULT_HOST,
+               mlops_port=Constants.MLOPS_DEFAULT_PORT,
+               user=Constants.MLOPS_DEFAULT_USER,
+               password=None):
+        """
+        Attach to a running ION and run in its context.
+        Side effect: sets up mlops_context
+        :param ion_id: the id of the ION to connect to
+        :param mlops_server: the host to connect to
+        :param mlops_port: the port MLOps is using
+        :param user: user name to use for connection
+        :param password: password to use for authentication, if None (default) then prompt for password
+        :return:
+    `
+        Note: Attach only works for pure python code
+        """
+        self._logger.info("Connecting to mlops: {} {}: {} user: {} pass: {}".format(
+            mlops_server, Constants.ION_LITERAL, ion_id, user, password))
+
+        if password is None:
```

Comment:
im not sure how this will work?

Reply:
Not sure which part is not clear. Anyway that is legacy code, I just move it from devel. 
The line above logs the info of elements we connect to. If the user didn't input a password yet then the password will be empty and the user will only input it in the line afterwards.

Reply:
we are interactively asking for a password - not sure how that works from an api point of view